plugins/legacy-plugin-chart-parallel-coordinates/package.json

```diff
@@ -28,8 +28,10 @@
     "access": "public"
   },
   "dependencies": {
+    "@superset-ui/style": "^0.14.3",
     "d3": "^3.5.17",
-    "prop-types": "^15.6.2"
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1"
```

Comment:
Same issue with other plugins.

Reply:
react and prop-types should also be peerDependencies.

Reply:
react is peer but prop-types is dep

Reply:
Is it really resolved? The code is still the same.